subprojects/scala/src/integTest/groovy/org/gradle/scala/compile/ZincScalaCompilerMultiVersionIntegrationTest.groovy

```diff
@@ -37,8 +37,8 @@ class ZincScalaCompilerMultiVersionIntegrationTest extends MultiVersionIntegrati
             task assertZincVersion {
                 dependsOn configurations.zinc
                 doLast {
-                    def resolvedDependencies = configurations.zinc.getResolvedConfiguration().getFirstLevelModuleDependencies()
-                    assert resolvedDependencies.find { it.moduleGroup == "org.scala-sbt" && it.moduleName == "zinc_2.13" && it.moduleVersion == "${version}" }
+                    def resolvedDependencies = configurations.zinc.incoming.dependencies
```

Comment:
I think this is the declared dependencies and not the resolved ones. I think you need to go through resolutionResult.  I don't think it's meaningful that we checked the first level dependencies.